Makefile

```diff
@@ -45,7 +45,7 @@ ui-install:
 
 .PHONY: ui-build
 ui-build:
-	cd $(UI_PATH) && npm run build
+	cd $(UI_PATH) && CI="" npm run build
```

Comment:
What's the reason behind this unsetting of the `CI` variable?

Reply:
it's because there are some warning raised coming from some dependencies in `node-modules`. I didn't find other way to ignore them than like that.
Like that warnings are not treated as errors